parser/yy_parser.go

```diff
@@ -328,6 +332,12 @@ func getUint64FromNUM(num interface{}) uint64 {
 		return uint64(v)
 	case uint64:
 		return v
+	case string:
+		n, err := strconv.Atoi(v)
+		if err != nil {
+			return 0
+		}
+		return uint64(n)
```

Comment:
I don't think it's a good idea to cast a string to an integer in the parser. I think storing the string in the AST and letting the SQL engine cast the value will be better. WDYT? @tangenta